### PR TITLE
Update "Import browser data" - When your imported data is sent to Microsoft

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:

Public rendered preview (approx layout):
https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/import-data/microsoft-edge/privacy-whitepaper/index.md#import-browser-data - starting at "Importing your data is completed locally", now a sep. para. leading into two list items.

Internal private preview (full layout); requires linked GitHub account per [Create and configure a GitHub account](https://review.learn.microsoft.com/help/get-started/setup-github) - 
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2934#import-browser-data - starting at "Importing your data is completed locally", now a sep. para. leading into two list items.

AB#47778992

### CODE REVIEW

**Review by amatharoo1459 — APPROVED** [review]
Reviewed text and links.

**Comment by mikehoffms** [comment]
This PR LGTM.

**Review by chgangwa — APPROVED** [review]
LGTM

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -432,7 +432,13 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
...
+With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.
+
+Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 
+
+* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.
+
+* Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.  To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).
```
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:441]: 1) Please add directions for how to opt-out of personalization on Bing.
2) For clarity, I suggest starting a new paragraph for how to view/delete imported data.
3) This link "[Microsoft Privacy Dashboard]" takes me to myaccount.microsoft.com and makes me resign into my account. Once i land on that page, it still is not obvious how to view/delete the imported data. I suggest using a more direct link to this data. I also suggest including step by step instructions to delete the data due to the ambiguity of the link.

```suggestion
* Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.  

To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).
```

**Review by apryzgoda — CHANGES_REQUESTED** [review]

File: microsoft-edge/privacy-whitepaper/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:441]: separated paragraphs

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -432,7 +432,21 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
...
+With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.
+
+Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 
+
+* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.
+
+* Allow personalization of experiences and advertising on [Bing](https://www.bing.com), Microsoft News, and other Microsoft services.
+<!-- todo: add directions for how to opt-out of personalization on Bing -->
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:442]: @amatharoo1459 - Add directions for how to opt out of personalization on Bing.  
per point #1 in [PR comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files/b1c6d7a11cca3887a87f9790abe864231fe1d6ca#r1410027587) from A. P.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 3]
```diff
@@ -432,7 +432,21 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
...
+
+To view or delete the imported browsing history that's available for personalization, along with Microsoft Edge browsing history, go to the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).
+<!-- todo:
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:445]: @amatharoo1459 - 
* Provide a more direct link to view/delete the imported data.
* Include step-by-step instructions to delete the data.

Do those b/c the existing link goes to myaccount.microsoft.com and requires re-sign in to my account.  At that page, it still is not obvious how to view/delete the imported data.

per Point #3 in [PR Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files/b1c6d7a11cca3887a87f9790abe864231fe1d6ca#r1410027587)

File: microsoft-edge/privacy-whitepaper/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md]: Dedicated [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files#r1410053855) for Point 1
Dedicated [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2934/files#r1410055033) for Point 3

File: microsoft-edge/privacy-whitepaper/index.md [hunk 4]
```diff
@@ -432,7 +432,16 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
...
+
+Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 
+
+* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.
+
+* In Microsoft Edge settings, turn on the toggle **Privacy, search, and services** > **Personalization & advertising** > **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news**.
+
+You can manage your browsing history by visiting [https://account.microsoft.com](https://account.microsoft.com).
+<!-- todo: clarify scope: locally there is one set, cloud there is another.  Dashboard has documentation about browsing history that talks about local vs. cloud data -->
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:444]: @amatharoo1459 - which browsing history? Locally has one browsing history; cloud has a distinct browsing history.  Check wording in Dashboard about browsing history that talks about local vs. cloud data.

File: microsoft-edge/privacy-whitepaper/index.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md]: see fresh Files Changed page

File: microsoft-edge/privacy-whitepaper/index.md (hunk 4 above)
Comment (amatharoo1459) [reply microsoft-edge/privacy-whitepaper/index.md:444]: image

File: microsoft-edge/privacy-whitepaper/index.md [hunk 5]
```diff
@@ -432,7 +432,16 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
...
+With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.
+
+Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 
+
+* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.
+
+* In Microsoft Edge settings, turn on the toggle **Privacy, search, and services** > **Personalization & advertising** > **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news**.
+
+You can manage your cloud browsing history by visiting [https://account.microsoft.com](https://account.microsoft.com).
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:443]:
```suggestion
You can manage your browsing history that's stored on secure Microsoft servers by visiting [https://account.microsoft.com](https://account.microsoft.com).
```

File: microsoft-edge/privacy-whitepaper/index.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md]: You can manage your browsing history that's stored on secure Microsoft servers by visiting [https://account.microsoft.com](https://account.microsoft.com).

File: microsoft-edge/privacy-whitepaper/index.md [hunk 6]
```diff
@@ -432,7 +432,15 @@ To control Image Enhancement, go to `edge://settings/privacy` and turn on or off
...
+With your confirmation, Microsoft Edge imports browser data from other browsers such as Google Chrome, Mozilla Firefox, or Internet Explorer. Microsoft Edge imports data from your most used browser as defined by your operating system.  If you choose to regularly import your browsing data, browsing data will be imported each time Microsoft Edge is launched.
+
+Importing your data is completed locally on your device and is stored locally.  The imported data is sent to Microsoft only after you do at least one of the following actions: 
+
+* Sign into Microsoft Edge with a Microsoft account, and turn on sync'ing of your browsing data across all signed-in devices.
+
+* In Microsoft Edge settings, turn on the toggle **Privacy, search, and services** > **Personalization & advertising** > **Allow Microsoft to save your browsing activity including history, usage, favorites, web content, and other browsing data to personalize Microsoft Edge and Microsoft services like ads, search, shopping and news**.
+
+You can manage your browsing history that's stored on secure Microsoft servers by visiting [https://account.microsoft.com](https://account.microsoft.com).
```
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:443]:
```suggestion
You can manage your browsing history that's stored on Microsoft servers by visiting [https://account.microsoft.com](https://account.microsoft.com).
```

**Review by apryzgoda — CHANGES_REQUESTED** [review]
Spoke with Andrea Chin to confirm support. Please remove the word "secure" from line 443. After that I will approve this.

File: microsoft-edge/privacy-whitepaper/index.md (hunk 6 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:443]: removed secure

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Champ Approved

**Pull request merged**